base/expr.jl

```diff
@@ -71,7 +71,7 @@ There is a difference between `@macroexpand` and `macroexpand` in that the `macr
 also takes a module where the expansion takes place.
 This is best seen in the following example:
 
-```jldoctest
+```julia-repl
```

Comment:
This is difficult to doctest now since one needs to get a reference to the outer module which Documenter creates itself.